quic/s2n-quic-core/src/recovery/bbr/full_pipe.rs

```diff
@@ -47,19 +47,36 @@ impl Estimator {
         &mut self,
         rate_sample: bandwidth::RateSample,
         max_bw: Bandwidth,
-        in_recovery: bool,
         ecn_ce_count_too_high: bool,
-        max_datagram_size: u16,
     ) {
         if self.filled_pipe {
             return;
         }
 
         self.filled_pipe = self.bandwidth_plateaued(rate_sample, max_bw)
-            || self.excessive_inflight(rate_sample, in_recovery, max_datagram_size)
             || self.excessive_explicit_congestion(ecn_ce_count_too_high);
     }
 
+    /// Called on each new loss round
+    ///
+    /// Excessive inflight is checked at the end of a loss round, not a regular round, as done
+    /// in tcp_bbr2.c/bbr2_check_loss_too_high_in_startup
```

Comment:
is it possible(easy enough) to add a debug assertion for this here?

Reply:
and or possibly replicate this comment in bbr/startup.rs

Reply:
I can't really add a debug assertion since the full_pipe::Estimator doesn't have access to the loss round counter, but I can put the comment in startup.rs